tests/inject/karma.conf.js

```diff
@@ -25,10 +46,6 @@ module.exports = (config) => {
                 rollupPluginTypescript({
                     typescript,
                     tsconfig: rootPath('tests/inject/tsconfig.json'),
-                    removeComments: false,
-                    sourceMap: true,
-                    inlineSources: true,
-                    noEmitOnError: true,
```

Comment:
Moved these to the tsconfig.json file because the options object doesn't include references to these.